database/mssql/scripts/dbscripts/PSP_PIMS_S15_00/Build/01_PSP_PIMS_S15_00_Build.sql

```diff
@@ -13241,3 +13241,146 @@ ALTER TABLE "dbo"."PIMS_WORKFLOW_MODEL"
 	ON DELETE NO ACTION 
 	ON UPDATE NO ACTION 
 GO
+
```

Comment:
@dfilteau please see additions in build script.
1) PIMS_TENANT (previously added by Jeremy, not tracked in your schema).
2) updated Contact manager view is this the right place to add it?